graphql/getter.js

```diff
@@ -60,6 +62,27 @@ export default class Getter {
     return this;
   };
 
+  withBm25 = (bm25Obj) => {
+    try {
+      this.bm25String = new Bm25(bm25Obj).toString();
+    } catch (e) {
+      this.errors = [...this.errors, e];
+    }
+
+    return this;
+  };
+
+  withHybrid = (hybridObj) => {
+    try {
+      this.hybridString = new Hybrid(hybridObj).toString();
```

Comment:
`this.hybridString` is never used, therefore it is not present in final request sent to server